tests/test_component/test_amgr.py

```diff
@@ -16,6 +16,37 @@
 MLAB = 'mongodb://entk:entk123@ds143511.mlab.com:43511/entk_0_7_4_release'
 
 def test_amgr_initialization():
+    amgr_name = ru.generate_id('test.appmanager.%(item_counter)04d', ru.ID_CUSTOM)
+    amgr = Amgr(hostname=hostname, port=port,name=amgr_name)
+
+    assert amgr._name.split('.') == amgr_name.split('.')
+    assert amgr._sid.split('.') == amgr_name.split('.')
+    assert amgr._uid.split('.') == ['appmanager', '0000']
```

Comment:
I think line 24-49 is duplicated. Do you think this is necessary? 
IMO, specifying amgr name only effects amgr._name and amgr._sid, so lines 22-23 should 
suffice. What do you think?

Reply:
I think we need to make sure that `__init__` runs correctly whether the name is setup or not. I can remove them, though